### PR TITLE
TP8 Documentation and Release Notes Updates

### PR DESCRIPTION
**Pull request by geekspertise**
Here are the updated docs for Service Mesh TP8 including the 3scale Istio Adapter 0.3 documentation update and some general clean up. Can @knrc @tvieira @JStickler @brian-avery and @vramosp provide feedback?

### CODE REVIEW

**Comment by knrc** [comment]
@geekspertise These changes look good to me, is there a preview version?

**Comment by geekspertise** [comment]
> @geekspertise These changes look good to me, is there a preview version?

Unfortunately the OpenShift Preview Bot doesn't seem like it was ever fixed to work with my account.

**Comment by tvieira** [comment]
LGTM

File: servicemesh-install/topics/appendix-threescale-adapter.adoc
Comment (JStickler) [inline servicemesh-install/topics/appendix-threescale-adapter.adoc]: Hybrid authentication method

**Review by JStickler — CHANGES_REQUESTED** [review]
Since we're not talking about a mobile app, all references in the 3Scale topic to "app" should be changed to "application".  Other than that, looks pretty good to me.

**Comment by geekspertise** [comment]
PR Update w/ fresh content: 
@knrc and I added a new topic comparing the upstream Istio community install versus Service Mesh. I also addressed the previous comments and a few other issues. @knrc and @JStickler can you provide feedback on the new content? And @vramosp please take a look at the 3scale Adapter docs as some language changes were made so it fits with the Service Mesh docs.

File: servicemesh-install/topics/install-remove.adoc
Comment (brian-avery) [inline servicemesh-install/topics/install-remove.adoc]: The following command removes...

...before running the following commands.

Seems repetitive.

The following command removes the {ProductShortName} from an existing installation. Provided you are logged in as a cluster admin, you can run these commands from any host with access to the cluster. ?

File: servicemesh-install/topics/servicemesh-overview.adoc
Comment (brian-avery) [inline servicemesh-install/topics/servicemesh-overview.adoc]: {ProductName} adds has two spaces.

File: servicemesh-install/topics/tutorial-prometheus.adoc
Comment (brian-avery) [inline servicemesh-install/topics/tutorial-prometheus.adoc]: Prometheus is an open source system

File: servicemesh-release-notes/topics/servicemesh-overview.adoc
Comment (brian-avery) [inline servicemesh-release-notes/topics/servicemesh-overview.adoc]: There are two spaces after {ProductName}

File: servicemesh-install/topics/servicemesh-istio-comparison.adoc
Comment (knrc) [inline servicemesh-install/topics/servicemesh-istio-comparison.adoc]: The automatic injection should refer to the annotation rather than configuration of the webhooks, IIRC we have a section in the documentation which has the text so we could refer to that section if necessary.

File: servicemesh-install/topics/servicemesh-istio-comparison.adoc
Comment (knrc) [inline servicemesh-install/topics/servicemesh-istio-comparison.adoc]: Sorry, there should also be a wildcard match in this list

exact, wildcard, prefix and suffix.

File: servicemesh-install/topics/servicemesh-istio-comparison.adoc
Comment (knrc) [inline servicemesh-install/topics/servicemesh-istio-comparison.adoc]: Should the comma be there after valid?

File: servicemesh-install/topics/servicemesh-istio-comparison.adoc
Comment (knrc) [inline servicemesh-install/topics/servicemesh-istio-comparison.adoc]: Perhaps include 'regular expression' in this?

File: servicemesh-release-notes/topics/rn-known-issues.adoc
Comment (JStickler) [inline servicemesh-release-notes/topics/rn-known-issues.adoc]: It looks like there is a workaround for this issue that should also be documented.

**Review by JStickler — COMMENTED** [review]
Just one update, other than that looks good to me.

**Comment by geekspertise** [comment]
@kalexand-rh and @vikram-redhat I got the blessing from @knrc to merge this content.

**Comment by vikram-redhat** [comment]
@geekspertise - I can't recall if we are going through the peer review process for ServiceMesh docs. If yes, can you put this on peer review (if it hasn't already been peer reviewed)?

**Comment by geekspertise** [comment]
It has already been peer reviewed.

**Comment by geekspertise** [comment]
@vikram-redhat When is this PR going to be merged? It was peer reviewed before I requested the merge.

**Pull request merged**
